tests/pytests/functional/pillar/test_gpg.py

```diff
@@ -10,7 +10,6 @@
 from saltfactories.utils.processes import ProcessResult
 
 pytestmark = [
-    pytest.mark.windows_whitelisted,
```

Comment:
Same here.

Reply:
GPG was not previously installed on Windows so these tests didn't run before (even though they were whitelisted). The tests don't work even though gpg is installed on the new AMI. Going back to skipping them.

Reply:
Ok